src/MeshIO.jl

```diff
@@ -16,6 +17,7 @@ include("io/ply.jl")
 include("io/stl.jl")
 include("io/obj.jl")
 include("io/2dm.jl")
+include("io/msh.jl")
 
 load(fn::File{format}, MeshType=GLNormalMesh) where {format} = open(fn) do s
     skipmagic(s)
```

Comment:
i take from ` add_format(format"MSH", (), ".msh")` that MSH doesn't have any magic bytes, so you shouldn't need to call `skipmagic`!